.eslintrc.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
         '@typescript-eslint/prefer-optional-chain': 'error',
         eqeqeq: ['error', 'always'],
         'unicorn/prefer-string-slice': 'error',
+        'unicorn/no-array-callback-reference': 'error',
```

Comment:
I am not sure that we need the such rule =/

@vladimirpotekhin what do you think about it ?

Reply:
I think the same. let's drop this rule)